back/app/Http/Resources/MessageResource.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class MessageResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id'        => $this->id,
+            'title'     => $this->title,
+            'content'   => $this->content,
+            'link'      => $this->link,
+            'created_at'=> $this->created_at,
+            'updated_at'=> $this->updated_at,
+            'deleted_at'=> $this->deleted_at,
```

Comment:
J'ai oublié de te demander de rajouter ça pour bien formatter les attributs *created_at* et *updated_at*

```suggestion
            'id'                => $this->id,
            'title'             => $this->title,
            'content'      => $this->content,
            'link'             => $this->link,
            'createdAt'  => $this->created_at->toDateTimeString(),
            'updatedAt' => $this->updated_at->toDateTimeString(),
            'deleted_at' => $this->deleted_at,
```